remoteworkitem/remoteworkitem_test.go

```diff
@@ -34,20 +27,20 @@ func provideRemoteData(dataURL string) ([]byte, error) {
 	return responseData, nil
 }
 
-func provideRemoteGithubDataWithAssignee() ([]byte, error) {
-	return provideRemoteData(GithubIssueWithAssignee)
+func provideRemoteGithubDataWithAssignee(input string) ([]byte, error) {
+	return provideRemoteData(input)
 }
 
-func provideRemoteJiraDataWithAssignee() ([]byte, error) {
-	return provideRemoteData(JiraIssueWithAssignee)
+func provideRemoteJiraDataWithAssignee(input string) ([]byte, error) {
+	return provideRemoteData(input)
 }
 
-func provideRemoteGithubDataWithoutAssignee() ([]byte, error) {
-	return provideRemoteData(GithubIssueWithoutAssignee)
+func provideRemoteGithubDataWithoutAssignee(input string) ([]byte, error) {
+	return provideRemoteData(input)
 }
 
-func provideRemoteJiraDataWithoutAssignee() ([]byte, error) {
-	return provideRemoteData(JiraIssueWithoutAssignee)
+func provideRemoteJiraDataWithoutAssignee(input string) ([]byte, error) {
+	return provideRemoteData(input)
```

Comment:
With such good comments in the test setup of what the URL should represent, I don't see the wrapper methods around provideRemoteData as very useful, specially since they don't actually do anything.